sw/sw.mk

```diff
@@ -61,9 +61,11 @@ CAR_ELFLOAD_BLOCKING_SAFED_SRC_C := $(CAR_SW_DIR)/tests/bare-metal/hostd/safed_o
 CAR_ELFLOAD_BLOCKING_SAFED_PATH := $(basename $(CAR_ELFLOAD_BLOCKING_SAFED_SRC_C))
 CAR_ELFLOAD_PULPD_SRC_C := $(CAR_SW_DIR)/tests/bare-metal/hostd/pulp-offload.c
 CAR_ELFLOAD_PULPD_PATH := $(basename $(CAR_ELFLOAD_PULPD_SRC_C))
+CAR_ELFLOAD_PULPD_INTF_SRC_C := $(CAR_SW_DIR)/tests/bare-metal/hostd/pulp-offload-intf.c
+CAR_ELFLOAD_PULPD_INTF_PATH := $(basename $(CAR_ELFLOAD_PULPD_INTF_SRC_C))
```

Comment:
Do you think that doing:
```
CAR_ELFLOAD_PULPD_SRC_C +=$(CAR_SW_DIR)/tests/bare-metal/hostd/pulp-offload-intf.c
```
Would work the same or might break? So we don't have to add dedicated variables for each new test we add (not sure it would work though).

Reply:
Ditto, we can append to the genric source variable `CAR_ELFLOAD_PULPD_SRC_C`

Reply:
I don't think it would work. `CAR_ELFLOAD_PULPD_SRC_C` (i.e. `$(CAR_SW_DIR)/tests/bare-metal/hostd/pulp-offload.c`) is used as an argument of `offload_tests_template`. We would need to add a second `for` loop there to check all the source files and each time execute the command `$(CHS_SW_CC) $(CAR_SW_INCLUDES) $(CHS_SW_CCFLAGS) -c $(3) -o $(4).$(basename $(notdir $(header))).car.o;  ` where `$(3)` would be `pulp-offload.c` and `pulp-offload-intf.c`.

Reply:
Yes, I agree. @alex96295 what do you think?